### PR TITLE
Clarified error message

### LINKED ISSUE #3223: `connect` error message should explain you can only connect to servers that are backdoored or adjacent servers

### PR DESCRIPTION
**Pull request by waffleattack**
Fixes #3223 

I didn't provide a gif as I'm unable to access the game currently, plus it's simply changing a message.

### CODE REVIEW

**Review by phyzical — APPROVED** [review]
lgtm

File: src/Terminal/commands/connect.ts [hunk 1]
```diff
@@ -36,7 +36,7 @@ export function connect(
       terminal.connectToServer(player, hostname);
       return;
     }
-    terminal.error(`Cannot directly connect to ${hostname}`);
+    terminal.error(`Cannot directly connect to ${hostname}. Make sure the server is backdoored or adjacent to your current Server`);
```
Comment (ApamNapat) [inline src/Terminal/commands/connect.ts:39]:
```suggestion
    terminal.error(`Cannot directly connect to ${hostname}. Make sure the server is backdoored and adjacent to your current Server`);
```

**Review by ApamNapat — CHANGES_REQUESTED** [review]

File: src/Terminal/commands/connect.ts (hunk 1 above)
Comment (incubusnb) [reply src/Terminal/commands/connect.ts:39]: A backdoored server doesn't have to be adjacent to your current server, so `or` would be preferred.

**Comment by hydroflame** [comment]
definitely `or`

File: src/Terminal/commands/connect.ts (hunk 1 above)
Comment (ApamNapat) [reply src/Terminal/commands/connect.ts:39]: Yeah, my bad. Mixed connecting and hacking in my head

**Pull request merged**